promql_http_api/api_response.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, url: str, *args, **kwargs):
         self.retries = kwargs.get('retries', http_retries)
         self.timeout = kwargs.get('timeout', 0)
         self.backoff = kwargs.get('backoff', http_backoff)
+        self.authorization_token = kwargs.get('authorization_token', "")
```

Comment:
I suggest naming the member 'authorization_header' instead of 'authorization_token'.
The HTTP Authorization header can include one of several authentication schemes. Token authentication is one of them. Since we allow this variable to include any string of any length, this variable can support all of those schemes. Naming it 'authorization_token' is misleading, since it may imply that it is used only for token-based authentication, and that the library will prepend the string with the 'token' schema - which it does not.

You can read more about HTTP authorization here:
[https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization)